maestro-studio/web/src/components/device-and-device-elements/ActionModal.tsx

```diff
@@ -263,8 +263,9 @@ export default function ActionModal({
           break;
         case "KeyC":
           if (
-            (isMac && e.metaKey) ||
-            (!isMac && e.ctrlKey && !e.altKey && !e.shiftKey)
+            ((isMac && e.metaKey) ||
+              (!isMac && e.ctrlKey && !e.altKey && !e.shiftKey)) &&
+            (!window || (window && window.getSelection()?.toString() === "")) // No text is selected
```

Comment:
Nit: This is a bit challenging to read -- could we simplify this logic somehow or add comments for each condition?

Reply:
Can you check now @axelniklasson ? Does this looks better?

```
case "KeyC":
          if (
            (isMac && e.metaKey) || // If mac - Command is pressed
            (!isMac && e.ctrlKey && !e.altKey && !e.shiftKey) // Or If not mac - Only control key is pressed
          ) {
            // If no text is selected
            if (window && window.getSelection()?.toString() === "") {
              e.preventDefault();
              if (typeof selectedCommand?.content === "string") {
                copyCommand(selectedCommand.content);
              }
            }
          }
          break;
```